### DISCUSSION TITLE
UI Bug: Plots from one of my plot_directories is not showing up in farming tab.

### DISCUSSION THREAD

**Discussion by ray-su** [guydavis/machinaris#198] [Q&A] [answered]
**Describe the bug**
I have 3 plot_directories,  only plots from my first 2 plot_directories are showing up in farming tab, the 3rd folder is missing.  
However,  by running `chia plots check`, all 3 folders and all my plots show up correctly. And I also confirm all my plots are being farmed. Therefore, I believe this is an UI bug.

**Comment by guydavis** [comment, marked as answer]
Good day, please ensure you have set your `plots_dir` environment variable to a colon-separated list of all 3 in-container paths as per the [directions in the wiki](https://github.com/guydavis/machinaris/wiki/FAQ#farming).

Then you may need to clean-up any out-of-date paths left behind from previous values of `plots_dir`:

```
chia show dirs
chia plots remove -d STALE_IN_CONTAINER_PATH
flax plots remove -d STALE_IN_CONTAINER_PATH
```
Repeat the remove lines until only valid in-container paths are shown by `chia show dirs`.

Hope this helps.  If this response answers your question, please mark the Discussion as Answered.

**Reply by ray-su** [reply]
I am very sorry 😅. Just noticed I forgot to add my 3rd folders to the plots_dir variable, I only did it for the first two.. Sorry about that
